guides/introduction/quick_start_guide.md

```diff
@@ -114,3 +114,180 @@ This will lead you to a web page with a search bar that will allow you to search
 <br>
 <br>
 <MovieGif />
+
+## Integrate to Your Project
+
+The only step missing now is adding the search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a developer tool that generates all the search components needed to start searching.
+
+[Instant MeiliSearch](https://github.com/meilisearch/instant-meilisearch) works on common front-end environments as: [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript), [React](https://github.com/meilisearch/meilisearch-react), and [Vue.js](https://github.com/meilisearch/meilisearch-vue).
+
+#### Let's Try!
+
+- Create an `html` file, for example, `index.html`.
+- Open it in a text editor (notepad, sublime text, VSC, or other).
+- Copy-paste any of the code examples bellow and save the file.
+- Open `index.html` in your browser (double click on it in your folder).
+
+_We use browser builds for ease of integration. It is possible to do this with `npm` or `yarn`. Please refer to [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch) for documentation._
+
+:::: tabs
+
+::: tab JavaScript
+
+The following code sample uses plain [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript).
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <div>
+      <div id="searchbox" focus></div>
+      <div id="hits"></div>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/@meilisearch/instant-meilisearch/dist/instant-meilisearch.umd.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/instantsearch.js@4"></script>
+    <script>
+        const search = instantsearch({
+            indexName: "movies",
+            searchClient: instantMeiliSearch(
+                "http://localhost:7700"
+            )
+            });
+
+            search.addWidgets([
+            instantsearch.widgets.searchBox({
+                container: "#searchbox"
+            }),
+            instantsearch.widgets.hits({
+                container: "#hits",
+                templates: {
+                item: `
+                    <div>
+                    <div class="hit-name">
+                        {{#helpers.highlight}}{ "attribute": "title" }{{/helpers.highlight}}
+                    </div>
+                    </div>
+                `
+                }
+            })
+            ]);
+
+            search.start();
+    </script>
+  </body>
+</html>
+```
+
+The code above comes in multiple parts:
+
+- The body part of the page adds both `searchbox` and `hits` elements.
```

Comment:
```
- The body part of the page adds both `searchbox` and `hits` elements.
```
Seems like a confusing way to frame it, since everything in this example is happening in the `<body>`.  Maybe this would be better:
```suggestion
- The first four lines of the `<body`> add both `searchbox and `hits` elements.
```